pkg/extension/sumologicextension/extension.go

```diff
@@ -39,6 +38,7 @@ type SumologicExtension struct {
 	baseUrl          string
 	conf             *Config
 	logger           *zap.Logger
+	credentials      CredsGetter
```

Comment:
```suggestion
	credentialsGetter      CredsGetter
```

Calling this `credentials` will make it very confusing IMHO